gammapy/stats/feldman_cousins.py

```diff
@@ -0,0 +1,436 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Provide a Feldman Cousins algorithm."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+import matplotlib.pylab as plt
+from scipy import stats
+
+__all__ = ['find_confidence_interval_gauss',
+           'find_confidence_interval_poisson'
+           'construct_confidence_belt_PDFs',
+           'get_upper_and_lower_limit',
+           'fix_upper_and_lower_limit',
+           'find_limit',
+           'find_average_upper_limit',
+           'construct_confidence_belt']
```

Comment:
@dlennarz – Do you think all 8 functions are commonly used by normal users?
Or could some be considered internal helper functions that don't need to be listed in the docs (trying to keep the `gammapy.stats` API small here)?